account-decoder/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub enum UiAccountData {
 
 impl From<Vec<u8>> for UiAccountData {
     fn from(data: Vec<u8>) -> Self {
-        Self::Binary(bs58::encode(data).into_string())
+        Self::Binary64(base64::encode(data))
```

Comment:
Lines 57-58 below need some attention with this change:
```
UiAccountEncoding::Binary => UiAccountData::Binary(bs58::encode(data).into_string()),
UiAccountEncoding::Binary64 => account.data.into(),
```

Reply:
🤦 good catch. I just removed the `From<...>` implementation to prevent that